src/services/groupService.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  addUsersToGroup,
+  createGroup,
+  deleteGroup,
+  getGroupById,
+  getGroups,
+  updateGroup,
+} from '../data-access/typeORMDataAccess';
```

Comment:
```suggestion
import * as GroupRepository from '../data-access/typeORMDataAccess';
```
And then below you will use `GroupRepository.createGroup`, for example, instead of just `createGroup`. It would add a bit more context to your code, so that it would be easy to understand which methods you're using.

Reply:
fixed